internal/db/db.go

```diff
@@ -93,13 +93,24 @@ func NewDb(dbUri string, authToken string) (*Db, error) {
 	if IsUrl(dbUrl) {
 		var validSqldUrl bool
 		if validSqldUrl, db.urlScheme = IsValidSqldUrl(dbUrl); validSqldUrl {
-			db.driver = libsql
-			db.sqlDb, err = sql.Open("libsql", dbUrl)
+			db.driver = libsqlDriver
+			var options []libsql.Option
+			if authToken != "" {
+				options = append(options, libsql.WithAuthToken(authToken))
+			}
+			if proxy != "" {
+				options = append(options, libsql.WithProxy(proxy))
+			}
+			connector, err := libsql.NewConnector(dbUrl, options...)
```

Comment:
nit: we're adding auth token twice. Line 86 and line 99

Reply:
good catch. I actually wanted to remove it but forgot. Thanks